tests/spread/cross-compile/environment/task.yaml

```diff
@@ -1,25 +1,41 @@
 summary: Test envvars related to architectures
 
 environment:
-  SNAP/build_for_single_arch: build-for-single-arch
   SNAP/build_for_multi_arch: build-for-multi-arch
+  SNAP/build_for_multi_arch_error: build-for-multi-arch-error
+  SNAP/build_for_multi_arch_triplet_error: build-for-multi-arch-triplet-error
+  SNAP/build_for_part_error: build-for-part-error
+  SNAP/build_for_part_triplet_error: build-for-part-triplet-error
+  SNAP/build_for_single_arch: build-for-single-arch
   SNAP/build_for_unknown_arch: build-for-unknown-arch
+  SNAP/build_for_unknown_arch_error: build-for-unknown-arch-error
+  SNAP/build_for_unknown_arch_triplet_error: build-for-unknown-arch-triplet-error
   SNAP/target_arch: target-arch
 
 restore: |
   cd "./snaps/$SNAP"
-  snapcraft clean
+  snapcraft clean --destructive-mode
   rm -f ./*.snap
 
 execute: |
   cd "./snaps/$SNAP"
 
+  # test error scenarios
```

Comment:
This seems like it would do better as a separate test (`environment-errors`?)

Reply:
Done.  I separated the passing and failing scenarios into 2 separate suites